docs/UserGuide.md

```diff
@@ -18,76 +18,135 @@ Here is a **quick** overview of your internship journey with LinkMeIn:
 LinkMeIn is more than just an application—it's your **personalized** partner, offering a 
 seamless experience that outclasses conventional tracking methods.
 
-Additionally, LinkMeIn combines the rapid **efficiency** of CLI with the visual clarity of GUI,
-offering a **tailored** experience for internship management that outpaces the generic
+Additionally, LinkMeIn combines the rapid **efficiency** of Command Line Interface (CLI) with the visual clarity of 
+Graphical User Interface (GUI), offering a **tailored** experience for internship management that outpaces the generic
 spreadsheets of Excel.
 
-# Navigating the user guide
-This User Guide provides you with all the information you need to fully utilize LinkMeIn. 
+## Introducing LinkMeIn's User Interface ##
+LinkMeIn's user interface (UI) is designed to be **simple** and **intuitive**. The UI consists of three main components:
+
+**1. Command Box** - Where you type in commands for LinkMeIn to execute.
+
+**2. Company Detail Panel** - Displays the details of the company that you are viewing.
```

Comment:
> Displays the details of a **specific** company

_Might not be viewing, could be adding and hence there's a default company there?_